sale_partner_insurance/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module add an insurance on partner and company, add automatically a line on sale order and
```

Comment:
Could you explain a little bit the context to use this module as it is not quite clear what the insurance is related to. Thanks

Reply:
@mathieudelva I wanted to say, functionally. What does the insurance for? What is it related to? This is not clear yet.